3rdparty/darknet/cmake/darknet.cmake

```diff
@@ -0,0 +1,149 @@
+cmake_minimum_required(VERSION 2.8)
```

Comment:
Could you please write description about this file?
This is to be copied to darknet direcotry as `CMakeLists.txt`, and
In order to build darknet with cmake, right?